### PR TITLE
Access Closure Compiler via HTTPS (Chromium issue #760416).

### PR DESCRIPTION
**Pull request by micolous**
Currently, the Closure Compiler web service is accessed via HTTP.  If the request is intercepted, the compiled Javascript may be replaced with something else.  This patch uses HTTPS instead, which on modern versions of Python also validates certificates.

Bug: https://bugs.chromium.org/p/chromium/issues/detail?id=760416

The second commit fixes an issue where the debug stanza would cause warnings and errors to be raised about the wrong file number.

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def print_errors(errors, js_files):
     if error['file'].lower().find('externs') >= 0:
       filename = error['file']
     else:
-      fileno = int(error['file'][6:])
+      fileno = int(error['file'][6:]) - 1 # Debug flag is file 0
```
Comment (MatrixFrog) [inline build.py]: This seems unrelated?

**Comment by beaufortfrancois** [comment]
This is now fixed in https://github.com/GoogleChromeLabs/text-app/pull/320/files
